main/options/options.cc

```diff
@@ -459,11 +459,23 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                "Errors not mentioned will be silenced. "
                                "This option can be passed multiple times.",
                                cxxopts::value<vector<int>>(), "errorCode");
+    options.add_options("dev")("isolate-error-code",
+                               "Error code to report. "
+                               "Errors not mentioned will be silenced. "
+                               "This option can be passed multiple times. "
+                               "An alternative to --error-white-list.",
```

Comment:
I wonder if it's better to cut this line and say under the entry for `--error-white-list` something like `Deprecated, renamed to --isolate-error-code`?

Reply:
I flipped between both these ways of doing it. I will gladly flip back. 😄

Reply:
No strong preference, personally---it's an improvement either way. :)

Reply:
I pushed a new commit for this. I also don't know what was up with the failing help output test - I thought I had the indentation right as I copied it straight from the locally built sorbet `--help`. Maybe it'll work now I've changed it again! 🤷🏻